nead/nead.py

```diff
@@ -119,3 +119,70 @@ def read(neadfile, MKS=None, multi_index=True, index_col=None):
     if('nodata' in ds.attrs.keys()): ds = ds.where(ds != ds.attrs['nodata'])
 
     return ds
+
+
+
+#%% Writes NEAD file (CSV file with NEAD formatted header)
+# Columns written in NEAD output will be the fields designated in the
+#   NEAD configuration file setting 'fields' in section 'FIELDS'
+# Paths must be absolute
+# Paths can use 'r' in front of strings with paths to designate them as a raw string if using Windows
+#   style paths with backslashes
+# Arguments:
+#   data_frame    Pandas dataframe
+#   nead_config   string with Path to NEAD configuration file, ex. r'C:\Users\Icy\gcnet\config\nead_header.ini'
```

Comment:
Is suggest changing the term "config" to "header" - that is what this is, the NEAD header. Calling it an "INI file" or "config file" is a bit confusing. There may actually be a NEAD config file one day (say a ~/.neadrc) that sets global NEAD behavior, like always enable MKS conversion, etc. 

This is for external(ish) facing terminology, like the doc string commented here. I'd also suggest making this change in the code, but it is not as important there.

I'm new to this GitHub review feature. Not sure how this comments gets handled..